src/auth/sso/BaseExternalProvider.php

```diff
@@ -334,7 +334,6 @@ protected function populateUser(User $user, array $data): User
     /**
      * Normalize a callback
      *
-     * @param mixed $callback
```

Comment:
This seems wrong? (And other changes like it)

Reply:
it already typed mixed, so `@param` is useless, except it has description

Reply:
@brandonkelly do you want me to skip these change? If yes, I can skip from config, eg: skip `MixedTypeRector`

Reply:
If it’s only removing `mixed` param docs, then yes. If we’re going to remove `@param` docs that don’t add any value over what’s in the code, we should be doing it for all types, not just `mixed`.